project.clj

```diff
@@ -1,7 +1,7 @@
 (def ks-version "1.2.0")
 (def tk-version "1.2.0")
 
-(defproject puppetlabs/trapperkeeper-metrics "0.1.3-SNAPSHOT"
+(defproject puppetlabs/trapperkeeper-metrics "0.2.0-SNAPSHOT"
```

Comment:
Ah, that might be true. My fear would be it would bump the `y` without setting the `z` back to zero, which would be bad...